tasks.py

```diff
@@ -121,55 +121,89 @@ def requirements_upgrade(ctx):
     ctx.run('pip-compile -U requirements.in')
 
 @task
-def build(ctx, ci=False):
-    'Build the dbt runner container'
+def build(ctx):
+    "Build the dbt runner container"
 
-    cache_from_arg = '--cache-from=dbt-runner' if ci else ''
-    ctx.run(f'docker build -t dbt-runner {cache_from_arg} -f Dockerfile .')
+    ctx.run(
+        f"docker-compose -f docker-compose.yml build dbt-runner"
+    )
```

Comment:
The `ci` flag was used in the github actions workflow to speed up testing so it didn't have to completely rebuild the container image every time.  I'd like to keep that.